src/features/DisplayControls/NursingTasks/components/NursingTasks.jsx

```diff
@@ -83,15 +83,19 @@ export default function NursingTasks(props) {
 
   useEffect(() => {
     const currentShift = shiftDetails.currentShiftHoursArray;
-    const firstHour = currentShift[0];
-    const lastHour = currentShift[currentShift.length - 1];
+    const [start, end] =
```

Comment:
I see some code duplication here. The same logic is being written in DrugChart Uitls. Is it possible to make a generic util method which can be called from both the files?

Reply:
Done as part of latest commit. Nice catch!!